src/main/java/com/treasuredata/client/TDClientHttpException.java

```diff
@@ -30,13 +30,13 @@
 {
     private final int statusCode;
 
-    private final Date retryAfter;
+    private final long retryAfter;
 
     public TDClientHttpException(ErrorType errorType, String message, int statusCode, Date retryAfter)
     {
         super(errorType, message);
         this.statusCode = statusCode;
-        this.retryAfter = retryAfter;
+        this.retryAfter = retryAfter == null ? -1 : retryAfter.getTime();
```

Comment:
[trivial] I prefer Long here to represent null to -1 although maybe it's just a matter of preference.